web-registry/src/App.tsx

```diff
@@ -137,49 +137,15 @@ const App: React.FC = (): JSX.Element => {
               element={<ProtectedRoute component={OrganizationProfile} />}
             />
             <Route
-              path="project-pages"
+              path="project-list"
               element={<ProtectedRoute component={ProjectList} />}
             />
-            <Route path="project-pages/:projectId" element={ProjectCreate}>
-              <Route
-                path="choose-credit-class"
-                element={<KeplrRoute component={ChooseCreditClassPage} />}
-              />
-              <Route
-                path="basic-info"
-                element={<KeplrRoute component={BasicInfo} />}
-              />
-              <Route
-                path="location"
-                element={<KeplrRoute component={ProjectLocation} />}
-              />
-              <Route path="story" element={<KeplrRoute component={Story} />} />
-              <Route
-                path="description"
-                element={<KeplrRoute component={Description} />}
-              />
-              <Route path="media" element={<KeplrRoute component={Media} />} />
-              <Route
-                path="metadata"
-                element={<KeplrRoute component={ProjectMetadata} />}
-              />
-              <Route path="roles" element={<KeplrRoute component={Roles} />} />
-              <Route
-                path="entity-display"
-                element={<KeplrRoute component={EntityDisplay} />}
-              />
-              <Route
-                path="review"
-                element={<KeplrRoute component={ProjectReview} />}
-              />
-              <Route
-                path="finished"
-                element={<KeplrRoute component={ProjectFinished} />}
-              />
-              <Route
-                path="edit"
-                element={<KeplrRoute component={ProjectEdit} />}
-              >
+            <Route path="project-pages">
+              <Route path=":projectId" element={<ProjectCreate />}>
```

Comment:
additional nesting here is also optional - I think it makes more sense in the long-term, but no need to add if you don't want.

Only required change to get the branch running is `element={<ProjectCreate />` (before, it was passing an uninstantiated object which is just an oversight on my part)